### ISSUE TITLE
Current implementation of Collections

### ISSUE THREAD

**Issue by hkdctol** [GSA/data.gov#1346]
## User Story

As Data.gov PMO, I want a document reflecting the current implementation of collections on Data.gov so that we can evaluate whether to enable as part of move to CKAN 2.8, or find another solution.

## Details

We have a unique implementation of collections and we need to evaluate options before we can move to CKAN 2.8.

## Acceptance Criteria
- [ ] Agreement within PMO that we have a current and comprehensive description of our current use of collections feature.

**Comment by jbrown-xentity** [comment]
We are on CKAN 2.10, this is no longer relevant.
